index.bs

```diff
@@ -2255,6 +2264,253 @@ Recording and Reporting Formal Objections</h3>
+	the entire [=TAG=] or [=AB=] is not expected or required to be dismissed.
+
+	An individual <em class=rfc2119>may</em> also <dfn lt="renounce | renunciation | renounces">renounce</dfn> their seat on the Council, for strong reason,
+	such as being forbidden by their employer to serve. The individual chooses the extent to which they explain
+	their renunciation.
+	Renunciation is disqualification from participation,
+	not abstention,
+	and <em class=rfc2119>should not</em> be used
+	to excuse an absence of participation.
+
+	Any person who has been [=dismissed=]
+	or who [=renounces=] their seat
+	does not receive [=Council=] materials,
+	take part in its deliberations,
+	help in the determination of consensus,
+	or vote.
+	The [=W3C Council=] <em class=rfc2110>may</em> still solicit and hear their testimony,
+	as they can of anyone else in the W3C community.
+
+<h5 id=council-chairing>
+Council Chairing</h5>
+
+	The <dfn lt="W3C Council Chair">Chair</dfn> of each W3C Council is chosen by its members,
+	by consensus if possible,
+	falling back to a vote if that fails.
+	The chair must be a member of that [=W3C Council=].
+
+<h5 id=council-deliberations>
+Council Deliberations</h5>
+
+	<p id=council-convention>
+	Upon appointment of the [=W3C Council Chair=]
+	and delivery of the [=Team=]’s report,
+	the [=W3C Council=] is considered to be convened
+	and can start deliberations.
+
+	Having reviewed the information gathered by the [=Team=],
+	the Council <em class=rfc2119>may</em> conduct additional research or analysis,
+	or request additional information or interviews from anyone,
+	including the Team.
+
+	The Council <em class=rfc2119>may</em>
+	further attempt to broker consensus,
+	which, if successful, disposes the formal objection.
+
+	Otherwise,
+	after sufficient deliberation,
+	the [=W3C Council=] decides whether to
+	<dfn>sustain</dfn> or <dfn>overrule</dfn> the objection.
+	When [=sustaining=] an objection,
+	it <em class=rfc2199>should</em> recommend a way forward.
+	The [=W3C Council=] <em class=rfc2119>may</em> [=overrule=] the [=Formal Objection=]
+	even if it agrees with some of the supportive arguments.
+
+	The Council <em class=rfc2119>may</em> form sub-groups for deliberation,
+	who may return with a recommendation,
+	but the full Council issues the final decision.
+	The decision of the [=W3C Council=] <em class=rfc2119>should</em> be unanimous,
+	and <em class=rfc2119>may</em> be issued under consensus.
+	However, if despite careful deliberation
+	the [=W3C Council=] is unable to reach consensus,
+	the [=W3C Council Chair=] may instead resort to voting.
+	In that case,
+	the decision is made by simple majority,
+	with the [=W3C Council Chair=] breaking any tie.
+	In case of a vote,
+	if two members of the Council who share the same affiliation cast an identical ballot,
+	then their ballots count as a one vote,
+	not two.
+
+	In the case of non-unanimous decisions,
+	members of the [=W3C Council=] who disagree with the decision
```

Comment:
```suggestion
	any member of the [=W3C Council=] who disagrees with the decision
```

Reply:
I think either phrasing would be correct, but the current one evokes more the idea that multiple people could write a single Minority Opinion whereas your proposed phrasing evokes more the idea that each person could write their own Minority Opinion. (Both phrasing allow either, but the emphasis is different.) I would lean towards the collective Minority Opinion since I think that would be a more natural way for the Council to lean.

Reply:
Even if it is subtle, I think it's worth pushing even a little bit for people to try and agree with each other. A single Minority Opinion can cite multiple arguments in any case.